G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/OutlookSaveAttachment/ShouldSaveAttachment_WhenSaveAttachmentCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,27 @@
+﻿♥outlookInboxFolder = ‴t.g1ant.test@gmail.com\Skrzynka odbiorcza‴
+♥toBeSentAttachmentPath = ‴♥environment⟦TEMP⟧\testattachmentsent.txt‴
+♥toBeDownloadedAttachmentPath = ‴♥environment⟦TEMP⟧\testattachmentdownloaded.txt‴
+
+outlook.open display false
+text.write text ‴Test text.‴ filename ♥toBeSentAttachmentPath
```

Comment:
It's good practice to move testing string to an variable, it decreases the possibility of mistake when writing the same string twice.